db/schema.rb

```diff
@@ -33,6 +33,18 @@
     t.index ["key"], name: "index_active_storage_blobs_on_key", unique: true
   end
 
+  create_table "admin_rate_changes", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb3", force: :cascade do |t|
```

Comment:
Curious as to why this is in your pull request

Reply:
Not sure.  I didn't touch any of that.  maybe it was in code i pulled...?  but that should have been in the main branch already, yeah?  is it causing any problems?

Reply:
Don't think it's really an issue but I was just curious.  I will proceed with merge